diff -Nru -Nru molly-guard-0.4.5.orig/profile.d/molly-guard.csh molly-guard-0.4.5/profile.d/molly-guard.csh
--- molly-guard-0.4.5.orig/profile.d/molly-guard.csh	1970-01-01 01:00:00.000000000 +0100
+++ molly-guard-0.4.5/profile.d/molly-guard.csh	2009-10-14 17:35:52.000000000 +0100
@@ -0,0 +1,12 @@
+
+##
+## debianize $PATH for use with molly-guard
+##
+
+# if we're superuser, reorder $PATH if it's wrong to get molly-guard working
+if ( `/usr/bin/id -u` == 0 ) then
+	# switch sbin entries
+	set path = ( `echo ${path} | sed -re 's#(^|\ )/sbin(\ |.*)/usr/sbin(\ |$)#\1/usr/sbin\2/sbin\3#'` )
+	# switch bin entries
+	set path = ( `echo ${path} | sed -re 's#(^|\ )/bin(\ |.*)/usr/bin(\ |$)#\1/usr/bin\2/bin\3#'` )
+endif
diff -Nru -Nru molly-guard-0.4.5.orig/profile.d/molly-guard.sh molly-guard-0.4.5/profile.d/molly-guard.sh
--- molly-guard-0.4.5.orig/profile.d/molly-guard.sh	1970-01-01 01:00:00.000000000 +0100
+++ molly-guard-0.4.5/profile.d/molly-guard.sh	2009-10-14 17:34:00.000000000 +0100
@@ -0,0 +1,14 @@
+
+##
+## debianize $PATH for use with molly-guard
+##
+
+# if we're superuser, reorder $PATH if it's wrong to get molly-guard working
+if [ `/usr/bin/id -u` -eq 0 ]
+then
+	# switch sbin entries
+	PATH=`echo ${PATH} | sed -re 's#(^|:)/sbin(:|.*)/usr/sbin(:|$)#\1/usr/sbin\2/sbin\3#'`
+	# switch bin entries
+	PATH=`echo ${PATH} | sed -re 's#(^|:)/bin(:|.*)/usr/bin(:|$)#\1/usr/bin\2/bin\3#'`
+	export PATH
+fi
